prebid-server/endpoints/openrtb2/pbs-endpoint-auction.md

```diff
@@ -195,27 +192,23 @@ it can be supplied
 
 ```
 {
-"ext": {
-  "prebid": {
-    "bidderparams": { 
-       "adapter1": { 
-          "key1": "data specific to key1", 
-          "key2": "  { 
-            "version": 3.3, 
-             "profileid": 1234 
-            } 
-          }, 
-      "adapter2": { 
-         "key3": "data specific to key3" 
-      }, 
-      "adapter3": { 
-          "key4": [ 
-            "data" 
-          ] 
-       } 
+  "ext": {
+    "prebid": {
+      "bidderparams": {
+        "bidderA": {
+          "key1": "some-value-1",
+          "key2": {
+            "version": 3.3,
+            "profileid": 1234
+          }
+        },
+        "bidderB": {
+          "key3": "some-value-3"
+        }
+      }
     }
   }
-} 
+}
```

Comment:
The example had invalid json. Fixed and aligned with our global "bidderA" naming approach.